orc8r/gateway/python/magma/magmad/rpc_servicer.py

```diff
@@ -254,6 +254,35 @@ async def enqueue_log_lines():
             except queue.Empty:
                 pass
 
+    def CheckStateless(self, _, context):
+        """
+        Check the stateless mode on AGW
+        """
+        logging.info("RPC: config command check")
+        status = self._service_manager.check_stateless_services()
+        logging.info("RPC: config response %s", status.name)
+        if status == ServiceManager._return_codes.STATEFUL:
+            return magmad_pb2.StatelessAgwMode(
+                    agw_mode=magmad_pb2.StatelessAgwMode.STATEFUL)
+        elif status == ServiceManager._return_codes.STATELESS:
+            return magmad_pb2.StatelessAgwMode(
+                    agw_mode=magmad_pb2.StatelessAgwMode.STATELESS)
+
+
+    def ConfigureStateless(self, request, context):
```

Comment:
This needs the `@return_void` decorator to annotate this handler as a function that does not return anything, if not the servicer will try to parse a non-existent response.